Gem/Core/Coordinator/WalletCoordinator.swift

```diff
@@ -155,32 +152,34 @@ struct WalletCoordinator: View {
     var body: some View {
         VStack {
             if let currentWallet = keystore.currentWallet {
-                MainTabView(
-                    model: .init(wallet: currentWallet),
-                    navigationStateManager: $navigationStateManager
-                )
-                .tint(Colors.black)
-                .alert(Localized.UpdateApp.title, isPresented: $updateAvailableAlertSheetMessage.mappedToBool()) {
-                    Button(Localized.Common.cancel, role: .cancel) { }
-                    Button(Localized.UpdateApp.action, role: .none) {
-                        UIApplication.shared.open(PublicConstants.url(.appStore))
+                LockScreenScene(model: LockSceneViewModel()) {
```

Comment:
```suggestion
                LockScreenScene(model: LockSceneViewModel()) {
```

can we move this model to the property?